### PR TITLE
feat(review): collapse the screenshots with no diffs

### LINKED ISSUE #88: Use the card to display the diffs
Collapse the card when the score is 0.

### PR DESCRIPTION
**Pull request by oliviertassinari**
capture d ecran 2017-02-26 a 22 46 17

Closes #88.

### CODE REVIEW

File: src/review/routes/build/BuildScreenshotItem.js [hunk 1]
```diff
@@ -100,4 +150,22 @@ export default recompact.compose(
   connect(state => ({
     screenshotsBucket: state.data.config.s3.screenshotsBucket,
   })),
+  recompact.withObs(({ props$ }) => {
```
Comment (neoziro) [inline src/review/routes/build/BuildScreenshotItem.js]: `recompact.withState('expandIn', 'onExpandIn', ({ screenshotDiff: { score } }) => score > 0)`

Comment (oliviertassinari) [inline src/review/routes/build/BuildScreenshotItem.js]: https://github.com/neoziro/recompact/blob/master/src/withState.js#L47

Comment (oliviertassinari) [inline src/review/routes/build/BuildScreenshotItem.js]: Ahhh, ça utilise `callOrUse`.

Comment (neoziro) [inline src/review/routes/build/BuildScreenshotItem.js]: @param {*|Function} initialState

**Pull request merged**